Composer/packages/extensions/visual-designer/src/utils/hooks.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { useContext, useState, useEffect } from 'react';
+import { useContext, useState, useEffect, useRef } from 'react';
+import { debounce } from 'lodash';
```

Comment:
We should use debounce package instead:

`import debounce from 'lodash.debounce'`